### PR TITLE
Add unit tests for `$assignees` function builtin

### PR DESCRIPTION
**Pull request by shay2025**
## Description

This pull request adds unit tests for `$assignees` function builtin.

## Related issue

*None*

## Type of change

Improvements (non-breaking change without functionality)

## How was this tested?

Ran `task test` command.

## Checklist

- [x] I have performed a self-review of my code
- [x] I have ran `task check -f` and have no issues

### CODE REVIEW

**Comment by marcelosousa** [comment]
@shay2025 there's a typo - `assignes` -> `assignees`

File: plugins/aladino/functions/assignees_test.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+	"github.com/reviewpad/reviewpad/v2/lang/aladino"
+	mocks_aladino "github.com/reviewpad/reviewpad/v2/mocks/aladino"
+	plugins_aladino "github.com/reviewpad/reviewpad/v2/plugins/aladino"
+	"github.com/stretchr/testify/assert"
+)
+
+var assignees = plugins_aladino.PluginBuiltIns().Functions["assignees"].Code
+
+func TestAssignees(t *testing.T) {
```
Comment (ferreiratiago) [inline plugins/aladino/functions/assignees_test.go:19]: We need to improve this test in order not to rely on the `getDefaultMockPullRequestDetails`.

For instance, if we remove `Assignees: []*github.User{{Login: github.String("jane")}}` from the mock, this test will continue to pass but it's not performing a proper test.

We need to mock our tests based on `getDefaultMockPullRequestDetails` but always changes whatever we are testing to make sure what when the test fails is for the right reason.

**Review by ferreiratiago — CHANGES_REQUESTED** [review]
The test needs to be refactored. Please have a look at the comment :)

**Comment by reviewpad-team** [comment]
**Reviewpad Report**

**:information_source: Messages**
* If you are adding or updating a built-in, please do not forget to update the [docs](https://github.com/reviewpad/docs)
* If you have added a new function or action do not forget to include it in the built-in list!

**Review by ferreiratiago — APPROVED** [review]
LGTM

**Pull request merged**